front/app/components/Form/index.tsx

```diff
@@ -278,7 +278,7 @@ const Form = memo(
     return (
       <Box
         as="form"
-        height="100%"
+        height="100vh"
```

Comment:
Is it ok to change this in this file, which is probably used in different parts of the application?

Reply:
Ah you're right! It changes it in the user profile settings when the _jsonforms_custom_fields_ feature flag is enabled. I'll fix this in another ticket. Thanks for catching it!

Reply:
Chatted quickly with Edwin - I'll implement all the fixes from these comments in my [PR which is currently open](https://github.com/CitizenLabDotCo/citizenlab/pull/2547) :+1: